src/libxrpl/protocol/LedgerFormats.cpp

```diff
@@ -364,6 +364,36 @@ LedgerFormats::LedgerFormats()
             {sfPreviousTxnLgrSeq,   soeREQUIRED}
         },
         commonFields);
+		
+	add(jss::MPTokenIssuance,
+        ltMPTOKEN_ISSUANCE,
+        {
+            {sfIssuer,               soeREQUIRED},
+            {sfSequence,             soeREQUIRED},
+            {sfTransferFee,          soeDEFAULT},
+            {sfOwnerNode,            soeREQUIRED},
+            {sfAssetScale,           soeDEFAULT},
+            {sfMaximumAmount,        soeOPTIONAL},
+            {sfOutstandingAmount,    soeREQUIRED},
+            {sfLockedAmount,         soeDEFAULT},
+            {sfMPTokenMetadata,      soeOPTIONAL},
+            {sfPreviousTxnID,        soeREQUIRED},
+            {sfPreviousTxnLgrSeq,    soeREQUIRED}
+        },
+        commonFields);
+
+    add(jss::MPToken,
+        ltMPTOKEN,
+        {
+            {sfAccount,           soeREQUIRED},
+            {sfMPTokenIssuanceID, soeREQUIRED},
+            {sfMPTAmount, soeDEFAULT},
+            {sfLockedAmount, soeDEFAULT},
+            {sfOwnerNode,            soeREQUIRED},
```

Comment:
`OwnerNode` is marked default in the spec.

Reply:
As above, IMHO, the spec should be updated for `sfOwnerNode`. There is always an owner, so the field should be required, as it is for other objects.

Reply:
Yes this is a mistake in the spec